integration/client/container_linux_test.go

```diff
@@ -667,13 +659,6 @@ func testContainerUser(t *testing.T, userstr, expectedOutput string) {
 func TestContainerAttachProcess(t *testing.T) {
 	t.Parallel()
 
-	if runtime.GOOS == "windows" {
```

Comment:
just out of curiosity, how did you know this was no longer an issue? Is it related to Go 1.23?

Reply:
From the file name `*linux_test.go` will be included in test only if platform is linux, the updated static check linter pointed it out.

Reply:
oh nice :) I definitely didn't catch that :)